packages/react-core/src/components/CalendarMonth/examples/CalendarMonth.md

```diff
@@ -17,3 +17,14 @@ beta: true
 
 ```ts file="./CalendarMonthSelectableDate.tsx"
 ```
+
+### Date range
+
+In this example, there are two dates selected: the first is a range start date and the second is a range end date. As you hover over dates that take place _after_ the range start date, a style is applied to show which dates are included in that range. This style will persist as you continue to hover within the calendar, even after selecting a range end date.
```

Comment:
So I think we can leave the example as an end date picker.  I would improve the text her to hopefully alleviate some confusion.  I think we want to emphasis the fact that the example demonstrates selection of an end range.   Ideally we would link to the Date Pick demo that shows selection of both start and end date date range (we don't have a precedent for that though).  I would definitely remove the "adding the ability to update the range start date, is functionality that would need to be added manually."  I don't think that makes sense in the context of this example.

Reply:
I also think we should disable the dates before the start date.  I am not sure why we don't have disable functionality built right now (Probably an oversight when initially implemented).  The core example shows disabled dates.
https://www.patternfly.org/v4/components/calendar-month/html#range-end-date-selected-start-date-focused

Reply:
I opened this issue to address the disabled dates
https://github.com/patternfly/patternfly-react/issues/7124